lib/jdomodels/src/test/java/org/sagebionetworks/repo/model/dbo/dao/DBOActivityDAOImplTest.java

```diff
@@ -92,7 +92,6 @@ public void testLockActivityAndIncrementEtag() throws Exception {
 		
 		ChangeType type = ChangeType.UPDATE;
 		String returnedNewEtag = activityDao.lockActivityAndGenerateEtag(id.toString(), oldEtag, type);
-		verify(mockMessenger).sendMessageAfterCommit(eq(mockDbo), eq(type));
```

Comment:
We should not remove this, the WIKI object type has a SNS topic in the stack builder.

Reply:
This one still needed (activity)

Reply:
Right, wrong class :)